data/is_ocaml_yet/gui.md

```diff
@@ -0,0 +1,74 @@
+---
+id: gui
+question: Is OCaml GUI Yet?
+answer: Not quite yet, but we're getting there!
+categories:
+  - name: Frameworks
+    status: 🟡
+    description: |
+      These are libraries written entirely in OCaml. It has packages for syndicating content and working with RSS feeds.
+    packages:
+      - name: bogue
+      - name: lablgtk
+      - name: lablqml
+      - name: tsdl
+      - name: sfml
+  - name: 3d graphics
+    status: 🟡
+    description: |
+      These are libraries that are for manipulating 3D graphics. They offer a wide range of functionalities, including geometric transformations, rendering, shading, and handling user input.
+    packages:
+      - name: tgls
+      - name: glMLite
+      - name: irrlicht
+  - name: 2d graphics
+    status: 🟡
+    description: |
+      These libraries provides the necessary functionalities to create and manipulate 2D graphics.
+    packages:
+      - name: vg
+      - name: gg
+      - name: async_graphics
+      - name: cairo2
+  - name: Web (Js bindings)
+    status: 🟡
+    description: |
+      These libraries are used to create bindings between OCaml and JavaScript, allowing OCaml code to interact with JavaScript libraries. They help in creating web applications using OCaml by providing a bridge between OCaml and JavaScript.
+    packages:
+      - name: chartjs
```

Comment:
Doesn't this actually belong on the "Is OCaml Web Yet?" page?

Reply:
Agreed, let's remove this category.